pkg/api/v1alpha1/cluster_types.go

```diff
@@ -165,6 +165,7 @@ type ControlPlaneConfiguration struct {
 	// Taints define the set of taints to be applied on control plane nodes
 	Taints []corev1.Taint `json:"taints,omitempty"`
 	// Labels define the labels to assign to the node
+	// NOTE: CAPC provider supports using 'ds.meta_data.failuredomain' as label value and replaces it with proper failuredomain value.
```

Comment:
Can you add an e2e test case for this contract change?

Reply:
I agree, do we have a EKA-A doc somewhere I can update.